kuma/static/styles/minimalist/structure/_article.scss

```diff
@@ -20,5 +23,24 @@ Main wiki article content
         margin-top: 32px;
         font-size: 0.88889rem;
         color: $grey;
+
+        header {
+            @include visually-hidden;
+        }
```

Comment:
The way this is written we do not actually need the class `visually-hidden` on the `header` element. I am in two minds to whether we should just remove the class from the header, or change this to `.visually-hidden` 🤔 

Having the class on the `header` element makes the intent explicitly clear by just looking at the HTML. On the other hand, if you highlight the element in devtools and look at the CSS, it will also be clear what is happening. 🤔 

Perhaps doing something like:

```
header.visually-hidden {
  @include visually-hidden;
}
```

What do you think @joedarc?

Reply:
> Having the class on the header element makes the intent explicitly clear by just looking at the HTML.

@schalkneethling, I definitely agree. It's minor but it makes more sense when inspecting the HTML. Made the requested change. 👍